web/pages/signin.js

```diff
@@ -43,112 +43,138 @@ export default function SignIn() {
 			body: JSON.stringify(body),
 		});
 
-		res = await res.json();
-		if (res.message == 'Login Success!') {
+		if (res.status === 200) {
 			setTimeout(() => {
 				Router.push('/profile');
 			}, 1000);
 		}
 		setResponse(res);
 	}
 
-	return (
-		<Flex height="100vh" align="center" justify="center" backgroundColor="gray.700">
-			<VStack
-				className="noselect"
-				background="gray.100"
-				p="3rem"
-				align="center"
-				justify="center"
-				spacing="1rem"
-				color="black"
-			>
-				<Heading>Sign In</Heading>
+	if (user)
+		return (
```

Comment:
Ideally we only want one return statement to render a component. We can conditionally render this using the `{user && (<content>)}` syntax like we used with the response.